runway/module/serverless.py

```diff
@@ -172,6 +172,8 @@ def run_serverless(self, command='deploy'):
         sls_env_file = get_sls_config_file(self.path,
                                            self.context.env_name,
                                            self.context.env_region)
+        if self.options.get('options', {}).get('config', {}):
+            sls_opts.extend(['--config', self.options.get('options', {}).get('config', {})])
```

Comment:
This conditional check should be changed to only add `--config VAL` if the config option is provided

Reply:
I think the current implementation matches the behavior you are describing, I tested it locally and the resulting command ignores de --config parameter when it's not specified

Reply:
this looks to be correct its just difficult to read. maybe something like this would be easier?

```suggestion
        if self.options.get('options', {}).get('config'):
            sls_opts.extend(['--config', self.options['options']['config']])
```

Reply:
@ITProKyle , makes sense, thanks for the suggestion!, I'll update it accordingly